alertmanager/router.go

```diff
@@ -0,0 +1,47 @@
+package alertmanager
+
+import (
+	"sync"
+
+	"github.com/weaveworks/mesh"
+)
+
+// GossipRouter is the interface we use for a mesh router.
+type GossipRouter interface {
```

Comment:
Forgot: this type and its methods are only used within this package, so un-export them.